SafetyFirst/Numbers.fs

```diff
@@ -4,16 +4,52 @@ namespace SafetyFirst.Numbers
 /// An integer >= 0
 /// </summary>
 [<Struct>]
-type NaturalInt = private NaturalInt of int
+type NaturalInt = 
+  private | NaturalInt of int
 
-[<Struct>]
-type NegativeInt = private NegativeInt of int
+  static member zero = NaturalInt 0
+
+  member this.Value = let (NaturalInt t) = this in t
+  static member value (t:NaturalInt) = t.Value
+  static member op_Implicit (t:NaturalInt) = t.Value
+
+  member this.Increment = let (NaturalInt t) = this in PositiveInt (t + 1)
+  static member increment (t:NaturalInt) = t.Increment
```

Comment:
Some of the `.increment`, `.decrement`, and `.opposite` methods are potentially subject to integer overflow, resulting in a Numbers type that doesn't match its assumption (like PositiveInt -2147483648). 

The exceptions are `PositiveInt.decrement` and `PositiveInt.opposite`.